lib/deploy.js

```diff
@@ -165,30 +175,34 @@ Deploy.prototype.execute_cmds = function(cmds) {
 
 Deploy.prototype.generate_provider_file = function() {
   var result = "";
-  result = "web3.setProvider(new web3.providers.HttpProvider('http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort + "'));";
-  result += "web3.eth.defaultAccount = web3.eth.accounts[0];";
-
+  result = "var Web3 = require('web3'); var web3 = new Web3;";
+  result += "web3.setProvider(new web3.providers.HttpProvider('http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort + "'));";
```

Comment:
it's all in the blockchain config file as per the previous comment I have made :)